core/src/main/java/org/infinispan/statetransfer/StateProviderImpl.java

```diff
@@ -90,6 +99,15 @@
     */
    private final Map<Address, List<OutboundTransferTask>> transfersByDestination = new HashMap<>();
 
+   /**
+    * Flags used when requesting the local publisher for the entries.
+    */
+   private static final long STATE_TRANSFER_ENTRIES_FLAGS = EnumUtil.bitSetOf(
+         SKIP_OWNERSHIP_CHECK,
+         // Indicate the command to not use shared stores.
+         STATE_TRANSFER_PROGRESS
+   );
```

Comment:
@wburns  from [previous comments](https://github.com/infinispan/infinispan/pull/9985#discussion_r831632918). I added the new flag but kept the `SKIP_OWNERSHIP_CHECK` only because `DenyReadWriteRemoveAllTest` started to fail without it. Do you think keeping this flag here makes any sense?

Reply:
absolutely